src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/ParentOrchestrationRouteTest.java

```diff
@@ -89,6 +114,8 @@ public void testParentOrchestration() throws Exception {
 
         setSourceData(file);
         camelContext.getGlobalOptions().put(MappingConstants.ORCHESTRATED_ROUTE, JUDICIAL_USER_PROFILE_ORCHESTRATION);
+        camelContext.getGlobalOptions().put(uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.SCHEDULER_START_TIME, uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.getCurrentTimeStamp().toString());
+        camelContext.getGlobalOptions().put(uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.SCHEDULER_NAME, schedulerName);
         parentRoute.startRoute();
```

Comment:
you can move this code to a separate method and add annotation @before to avoid repeating code in every method